samples/wifi/wifi.rst

```diff
@@ -0,0 +1,10 @@
+.. _wifi-samples:
```

Comment:
You reference this doc as having more build and run details, but there are none here.